src/engine.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+/* eslint-env worker */
+import type { SubView } from "autolayout";
+import type ConstraintBuilder from "./constraint-builder";
+import { View } from "autolayout";
+
+// TODO: find the right webpack configuration to make
+// the worker code standalone from the bundle.
```

Comment:
:+1: – I've added it in a PR to the Formidable Practices document.
https://github.com/FormidableLabs/practices/pull/5